# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../tcp_wrappers/0018-tcp_wrappers-7.6-man-fromhost.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN tcp_wrappers_7.6.orig/hosts_access.3 tcp_wrappers_7.6/hosts_access.3
--- tcp_wrappers_7.6.orig/hosts_access.3	2004-04-25 00:10:48.000000000 +0200
+++ tcp_wrappers_7.6/hosts_access.3	2004-04-25 00:09:36.000000000 +0200
@@ -14,6 +14,9 @@
 struct request_info *request_set(request, key, value, ..., 0)
 struct request_info *request;
 
+void fromhost(request)
+struct request_info *request;
+
 int hosts_access(request)
 struct request_info *request;
 
@@ -60,6 +63,7 @@
 is available, host names and client user names are looked up on demand,
 using the request structure as a cache.  hosts_access() returns zero if
 access should be denied.
+fromhost() must be called before hosts_access().
 .PP
 hosts_ctl() is a wrapper around the request_init() and hosts_access()
 routines with a perhaps more convenient interface (though it does not
